packages/strapi-builder/index.ts

```diff
@@ -0,0 +1,50 @@
+import { Strapi, Plugin, Model, ComponentModel } from "strapi-types";
+
+export class StrapiBuilder {
+  strapi = new Strapi();
+  addContentType = (contentType: { [key: string]: any }) => {
```

Comment:
if in Strapi class somebody will add ContentType type, it will fails, at least create an alias for any

Reply:
fixed